tests/test_feature_aggregate.py

```diff
@@ -171,6 +171,7 @@ def test_aggregate_skip_intermediate(time_col):
     df = IamDataFrame(data, model="model_a", scenario="scen_a", region="World")
     df2 = df.rename(scenario={"scen_a": "scen_b"})
     df2.data.value *= 2
```

Comment:
I noticed that this is an artifact from before the refactoring of the timeseries-data backend. The current line does not have any effect.
```suggestion
    df2._data *= 2
```